frontend/taipy-gui/src/components/Taipy/TableFilter.tsx

```diff
@@ -114,22 +118,22 @@ const getFilterDesc = (
                         ? colType === "number"
                             ? parseFloat(val)
                             : colType === "boolean"
-                            ? val === "1"
-                            : colType === "date"
-                            ? getDateTime(val)
-                            : val
+                                ? val === "1"
+                                : colType === "date"
+                                    ? getDateTime(val)
+                                    : val
                         : val,
                 type: colType,
                 matchCase: !!matchCase,
             } as FilterDesc;
         } catch (e) {
-            console.info("could not parse value ", val, e);
+            console.info("Could not parse value ", val, e);
         }
     }
 };
 
 const FilterRow = (props: FilterRowProps) => {
-    const { idx, setFilter, columns, colsOrder, filter } = props;
+    const { idx, fieldHeader, fieldHeaderTooltip="", filter, columns, colsOrder, setFilter } = props;
```

Comment:
why `fieldHeaderTooltip=""` here and not in sort ?

Reply:
It's wrong.